src/styles/blocks/product.scss

```diff
@@ -0,0 +1,26 @@
+.product {
+  &__image {
+    margin-bottom: 32px;
+    width: 100%;
+  }
+
+  &__title {
+    margin-bottom: 8px;
+    font-size: 24px;
+    font-weight: 700;
+    line-height: 140%;
```

Comment:
you have lots of repetitions, use `mixin` like this: `@incluede title-style(24px, 140%, 700)`
`@mixin title-style($size, $height, $weight) {
  font-size: #{$size};
  line-height: #{$height};
  font-weight: #{$weight};
}`